packages/datetime/src/dateInput.tsx

```diff
@@ -98,6 +98,12 @@ export interface IDateInputProps extends IDatePickerBaseProps, IProps {
     outOfRangeMessage?: string;
 
     /**
+     * Whether the date popover should be inline.
+     * @default true
+     */
+    popoverInline?: boolean;
```

Comment:
This makes sense given the `popoverPosition` precedent, and I'm fine with this through that lens. But FYI @jchomicki we've discussed exposing all of `popoverProps` both here and on `DateRangeInput`; that prop would allow you to configure your popover however you want. @giladgray what do you think of jumping straight to that in this PR rather than adding another popover prop separately?

^ @jchomicki what I suggest above would admittedly be a slower, more annoying change, because you'd have to make sure you spread the `popoverProps` in the correct place so as to not allow overriding certain props:

```
             <Popover
                  autoFocus={false}
                  // where
                  content={popoverContent}
                  // does
                  enforceFocus={false}
                  // the
                  inline={true}
                  // spread
                  isOpen={this.state.isOpen && !this.props.disabled}
                  // go?
                  onClose={this.handleClosePopover}
                  popoverClassName="pt-dateinput-popover"
                  position={this.props.popoverPosition}
             >
```

That said, we may just want to wait for 2.0. Happy to hear thoughts.